functions/scripts/send-mass-text.js

```diff
@@ -0,0 +1,135 @@
+const functions = require('firebase-functions');
+const _ = require('lodash');
+const twilio = require('twilio');
+const Airtable = require('airtable');
+
+const twilioClient = twilio(functions.config().twilio.sid, functions.config().twilio.auth_token);
+
+const airtable = new Airtable({
+  apiKey: functions.config().airtable.api_key,
+});
+
+const base = airtable.base(functions.config().airtable.base_id);
+
+const { INTAKE_TABLE, VOLUNTEER_FORM_TABLE } = require('../airtable');
+
+const ENGLISH = 'English';
+const SPANISH = 'Spanish';
+
+const LANGUAGES = [ ENGLISH, SPANISH ];
+
+// Numbers in airtable may not be in a standard format
+// or may have other text in the field
+// assume all numbers are US numbers
+function normalizeRecords(defaultLanguage) {
+  return (record) => {
+    const number = record.get('Phone Number') || '',
+          digits = number.replace(/\D/g, ''),
+          formatted = digits.length === 10 ? '+1' + digits : null;
+
+    return {
+      phoneNumber: formatted,
+      language: record.get('Language') ? record.get('Language')[0] : defaultLanguage,
+    };
+  };
+}
+
+async function getPhoneNumbers(table, fields, defaultLanguage) {
+  const records = await base(table).select({
+    fields: fields
+  }).all();
+
+  return records.map(normalizeRecords(defaultLanguage))
+    .filter((record) => { return record.phoneNumber !== null; });
+}
+
+// Takes an array of languages that we will be using for this
+// message send. If an airtable record has no language preference
+// or a language preference other than the languages in this list,
+// we will group it with the default language: languages[0]
+async function getPhoneNumbersByLanguage(languages) {
+  const defaultLanguage = languages[0];
```

Comment:
nit: having the caller pass in a default language might make it harder to mess up